cmd/gerrit/gerrit.go

```diff
@@ -648,6 +650,26 @@ func (j *DSGerrit) ConvertDates(ctx *shared.Ctx, review map[string]interface{})
 						}
 					}
 				}
+				if patchTS.IsZero() {
+					continue
+				}
+				iComments, ok := shared.Dig(patch, []string{"comments"}, false, true)
+				if ok {
+					comments, ok := iComments.([]interface{})
+					if ok {
+						for commentNum, iComment := range comments {
+							comment, ok := iComment.(map[string]interface{})
+							if !ok {
+								continue
+							}
+							comment["level"] = "patchset"
+							comment["patchset_comment_num"] = fmt.Sprintf("%d:%d", patchNum, commentNum)
```

Comment:
Suggestion: `patchset_comment_num`  can be `patchset_comment_index` ?

Reply:
Changing.

Reply:
Updated.